src/pages/settings/user/common/hooks/useUserDetailsTabs.tsx

```diff
@@ -16,26 +16,26 @@ export function useUserDetailsTabs() {
   const { t } = useTranslation();
 
   const tabs: Tab[] = [
-    { name: t('details'), href: '/settings/user_details' },
+    { name: t('details'), href: route('/settings/user_details') },
```

Comment:
After pushing last changes on this PR, checker from the GitHub gave me an error about no-used import route, so to fix it I just added this import on the right place.